src/core/annotation.js

```diff
@@ -1895,47 +1895,45 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
       );
     }
 
+    // The annotationStorage can be null if no change occured on the form
+    // but a checkbox could be already checked when it's open.
```

Comment:
Sorry, but I'm finding this second line of the sentence slightly difficult to understand. Could it perhaps be re-worded to improve the clarity somewhat?